stories/topics/proc-gcp-generate-playbook.adoc

```diff
@@ -0,0 +1,48 @@
+[id="proc-gcp-generate-playbook"]
+
+= Generating the playbook
+
+To generate the playbook, run the command generator to generate the CLI command.
+
+[source,bash]
+----
+docker run --rm -v /tmp:/data $IMAGE command_generator gcp_setup_logging_monitoring \
+--data-file /data/logging-monitoring.yaml
+----
+
+Provides the following command:
+
+[source,bash]
+----
+docker run --rm --env PLATFORM=GCP -v /path/to/credentials:/home/runner/.gcp/credentials:ro \
+--env ANSIBLE_CONFIG=../gcp-ansible.cfg --env DEPLOYMENT_NAME=mu-deployment \
+--env GENERATE_INVENTORY=true  $IMAGE redhat.ansible_on_clouds.gcp_setup_logging_monitoring \
+-e 'gcp_deployment_name=mu-deployment \
+gcp_service_account_credentials_json_path=/home/runner/.gcp/credentials  components=["hubs","controllers"]\
+monitoring_enabled=True logging_enabled=True default_collector_interval=60s'
+----
+
+Run the supplied command to run the playbook.
+
+[source,bash]
+----
+$ docker run --rm --env PLATFORM=AWS -v ~/.aws/credentials:/home/runner/.aws/credentials:ro\
```

Comment:
Not sure where this example came from but it's AWS remove extension nodes, not GCP setup logging monitoring